src/data/real/nnreal.lean

```diff
@@ -725,6 +725,82 @@ end inv
 @[simp] lemma abs_eq (x : ℝ≥0) : |(x : ℝ)| = x :=
 abs_of_nonneg x.property
 
+section csupr
+
+variables {ι : Sort*} {f : ι → ℝ≥0}
+
+lemma mul_csupr (hf : bdd_above (set.range f)) (a : ℝ≥0) :
+  a * (⨆ i, f i) = ⨆ i, a * f i :=
+begin
+  by_cases hι : nonempty ι,
```

Comment:
I recommend
```suggestion
  casesI is_empty_or_nonempty ι,
```
Or whatever the lemma is really called

Reply:
Thanks!